widget/CommonScreens.xml

```diff
@@ -40,28 +40,41 @@ under the License.
             </widgets>
         </section>
     </screen>
-
     <screen name="Ofbiz-tutorialCommonDecorator">
```

Comment:
do we really need to change the Ofbiz-tutorialCommonDecorator decorator

Reply:
Ofbiz-tutorialCommonDecorator screen serves as a custom decorator for my application. We could also create another screen to serve as a custom decorator. Hence, it is not necessarily required to change the Ofbiz-tutorialCommonDecorator.